markov/tmatrix.R

```diff
@@ -1026,7 +1028,7 @@ tmatrix_std <- function(bb_exp = bb_exp_lg,
 
 get_results <- function(bat, pitch, stad, home, temp, date,
                         away, inning = 1, pitch_ct = 0) {
-  
+
   # Create list of states to iterate over
   states <- list(State$new(top = away, inning = inning, pitch_number = pitch_ct), 
```

Comment:
What do you do with the variable states that you create here?  It looks like you recreate it a few hundred lines laters.  Can you delete the first version of "states" variable?